src/server/pkg/obj/amazon_client.go

```diff
@@ -223,6 +230,7 @@ func newWriter(client *amazonClient, name string) *amazonWriter {
 	}
 	go func() {
 		_, err := client.uploader.Upload(&s3manager.UploadInput{
+			ACL:             &uploadACL,
```

Comment:
:+1: I'm glad to see that this was as simple to implement as I had hoped. This will make more complicated S3 egress setups _much_ nicer, especially for any organization large enough to use multiple accounts for things like billing, per-department access control, etc.